packages/protocol/contracts-0.8/stability/interfaces/IFeeCurrencyAdapter.sol

```diff
@@ -2,22 +2,51 @@
 pragma solidity >=0.8.7 <0.8.20;
 
 interface IFeeCurrencyAdapter {
+  /**
+   * @return The address of the adapted token.
+   */
   function getAdaptedToken() external view returns (address);
 
+  /**
+   * @return The multiplier that should be used when upscaling and downscaling.
+   */
   function digitDifference() external view returns (uint96);
 
+  /**
+   * @return returns the amount that it's debited after calling debitGasFees().
+   */
   function debited() external view returns (uint256);
 
+  /**
+   * @return The name of the adapted token.
+   */
   function name() external view returns (string memory);
 
+  /**
+   * @return The symbol of adapted token.
+   */
   function symbol() external view returns (string memory);
 
+  /**
+   * @return Returns the decimals expected by the vm.
+   */
+  function decimals() external view returns (uint8);
+
+  /**
+   * @return same as decimals.
+   */
   function expectedDecimals() external view returns (uint8);
 
-  function decimals() external view returns (uint8);
 
+  /**
+   * @notice See IFeeCurrency.
```

Comment:
I would explicitly say that these are like the ones in IFeeCurrency, but using the number of decimals native to this adapter.